tests/contrib/operators/test_dynamodb_to_s3.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from multiprocessing import SimpleQueue
```

Comment:
Was there any reason why this is using something from Multiprocessing? Wouldn't a plain old-python list work just as well?

Reply:
Raised as https://github.com/apache/airflow/pull/6836  (so that we can pull this commit in to 1.10.7)

Reply:
No good reason. I think initially I was using multiprocessing to do the s3 upload but later decided not to and I guess the tests did not get updated.

Reply:
Cool! Figured it was something like, just wanted to check before we release 1.10.7 with this + the changed test to find out we'd missed something important.